CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
-  To migrate you'll need to remove any references to `govuk-grid-row` mixin and use `.govuk-grid-row` class instead.
+If you’re importing a specific path, add `govuk/` after 'govuk-frontend/`. For example, if you're importing the button component:
 
-  ([PR #1343](https://github.com/alphagov/govuk-frontend/pull/1343))
+```nunjucks
+import Button from 'govuk-frontend/govuk/components/button/button'
+```
 
-- Remove grid-width mixin
+##### Update and add data-module attributes
 
-  https://github.com/alphagov/govuk-frontend/pull/1090 deprecated this mixin
-  and left it in as an alias to govuk-grid-width.
+You do not need to do anything if you're using Nunjucks macros and the [`initAll`](https://github.com/alphagov/govuk-frontend/blob/268b1e1b962d4bd293c5681868a2119f8b4ece60/docs/installation/installing-with-npm.md#option-1-include-javascript) function.
 
-  To migrate you'll need to replace any reference to the `grid-width` mixin with `govuk-grid-width`
+If you are not using the Nunjucks macros, add a `govuk-` prefix to `data-module` attribute values. For example:
 
-  ([PR #1342](https://github.com/alphagov/govuk-frontend/pull/1342))
+```html
+<div class="govuk-accordion" data-module="govuk-accordion">
+...
+</div>
+```
 
-- Remove the deprecated `$class` parameter from the `@govuk-grid-column` mixin
+The [button](https://design-system.service.gov.uk/components/button/) and [details](https://design-system.service.gov.uk/components/details/) components now also use the `data-module` attribute for initialisation. If you are not using Nunjucks macros, add:
 
-  To migrate, you'll need to update any places in your code where you use the
-  `@govuk-grid-column` mixin.
+- `data-module="govuk-button"` to each `<button>` HTML tag
+- `data-module="govuk-details"` to each `<details>` HTML tag
 
-  If you are currently passing a class name (`$class: 'foo'`) you should remove
-  the `$class` argument and wrap your call to the mixin in the class name
-  instead:
+If you're using your own JavaScript code to initialise components, add a `govuk-` prefix to any selectors that find components.
 
-  ```
-  .foo {
-    @include govuk-grid-column(...)
-  }
-  ```
+[Pull request #1443: Ensure GOV.UK Frontend component selectors cannot conflict when initialised](https://github.com/alphagov/govuk-frontend/pull/1443)
 
-  If you are currently passing `$class: false` then you can just remove the
-  `$class` argument, as this is now the default behaviour.
+##### Update CSS class names
 
-  ([PR #1376](https://github.com/alphagov/govuk-frontend/pull/1376))
+You do not need to do anything if you're using Nunjucks.
 
-- Ensure GOV.UK Frontend component selectors cannot conflict when initialised
+If you're using HTML or custom JavaScript, change:
 
-  Components which have JavaScript functionality include a data attribute called `[data-module]`,
-  we have added the `govuk-` prefix as a namespace.
+- `js-character-count` to `govuk-js-character-count`
```

Comment:
Nit picky but for consistency should this say in the GOV.UK Frontend character count component

Reply:
Since there's no legacy character count component we thought the distinction was not necessary

Reply:
Yeah I didn't see this as legacy vs non-legacy thing, it just looked like something might have been missed.

But I don't feel that strongly about it.

Reply:
Will leave this up to @m-green to decide

Reply:
I feel it should be ok. If users should only replace this in certain files, we should definitely clarify. But if I understand it right, this is a simple search-and-replace for everywhere you have `js-character-count`?